src/components/Book.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ id, title, category }) => (
+const Book = ({
+  id, title, category, removeBook, book,
+}) => (
   <tr>
     <td>{id}</td>
     <td>{title}</td>
     <td>{category}</td>
+    <td><button type="button" onClick={() => removeBook(book)}>Remove Book</button></td>
   </tr>
 );
 
 Book.propTypes = {
   id: PropTypes.number.isRequired,
   title: PropTypes.string.isRequired,
   category: PropTypes.string.isRequired,
+  removeBook: PropTypes.func.isRequired,
+  book: PropTypes.instanceOf(Object).isRequired,
 };
```

Comment:
If you decide to do that ☝️, don't forget to update your props validation as well.